Userland/Libraries/LibGUI/Variant.h

```diff
@@ -118,10 +123,16 @@ class Variant {
         return m_value.as_i64;
     }
 
-    unsigned as_uint() const
+    unsigned as_u32() const
     {
-        VERIFY(type() == Type::UnsignedInt);
-        return m_value.as_uint;
+        VERIFY(type() == Type::UnsignedInt32);
+        return m_value.as_u32;
+    }
+
+    unsigned as_u64() const
```

Comment:
unsigned for both u32 and u64... This doesn't seem right?

Reply:
Whoops, must've missed that one....